src/py-opentimelineio/opentimelineio/plugins/manifest.py

```diff
@@ -28,11 +28,17 @@
 import logging
 import os
 
-# on some python interpreters, pkg_resources is not available
-try:
-    import pkg_resources
-except ImportError:
+# pkg_resources can have bad performance characteristics in some circumstances
```

Comment:
clarity/punctuation:

In some circumstances pkg_resources has bad performance characteristics. Using this flag: OTIO_DISABLE_PKG_RESOURCE_PLUGINS disables OpenTimelineIO's import and of use of the pkg_resources system.